packages/graphback-datasync/src/DatasyncPlugin.ts

```diff
@@ -76,13 +77,13 @@ export class DatasyncPlugin extends GraphbackPlugin {
 
 
             // Diff queries
-            if (this.pluginConfig.generateDeltaQueries) {
-                const diffQuery = `${model.graphqlType.name}Delta`
+            if (parseMarker('delta', model.graphqlType.description)) {
+                const diffQuery = `get${model.graphqlType.name}Delta`
                 schemaComposer.Query.addFields({
-                    [diffQuery]: `[${model.graphqlType.name}]!`
-                })
+                    [diffQuery]: `[${model.graphqlType.name}]`
+                });
                 schemaComposer.Query.addFieldArgs(diffQuery, {
-                    lastSync: 'String!'
+                    lastSync: 'String'
```

Comment:
This is where it adds delta queries to the schema, I am going to replace the resolver Type with something more involved, `ResultList` type is a candidate for this but I think it wouldn't suffice as it does not have a `lastSync`

Reply:
This is done